<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>122</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>591</width>
    <height>122</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>591</width>
    <height>122</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Выдача рецепта</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>401</width>
      <height>91</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>401</width>
      <height>91</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter_5">
       <property name="minimumSize">
        <size>
         <width>399</width>
         <height>89</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>399</width>
         <height>89</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="minimumSize">
         <size>
          <width>399</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>399</width>
          <height>42</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>399</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>399</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Введите ФИО и дату рождения пациента</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="patientData">
         <property name="minimumSize">
          <size>
           <width>399</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>399</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_4">
        <property name="minimumSize">
         <size>
          <width>399</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>399</width>
          <height>42</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter_2">
         <property name="minimumSize">
          <size>
           <width>167</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>167</width>
           <height>42</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>167</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>167</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>Срок действия рецепта (мес.)</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox">
          <property name="minimumSize">
           <size>
            <width>167</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>167</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_3">
         <property name="minimumSize">
          <size>
           <width>227</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>227</width>
           <height>42</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>227</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>227</width>
            <height>15</height>
           </size>
          </property>
          <property name="text">
           <string>Введите название для файла с рецептом</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="filename">
          <property name="minimumSize">
           <size>
            <width>227</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>227</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>160</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>111</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>111</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QDateEdit" name="dateEdit">
       <property name="minimumSize">
        <size>
         <width>158</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>158</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="minimumSize">
        <size>
         <width>158</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>158</width>
         <height>23</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
